packages/wp-now/src/wp-now.ts

```diff
@@ -213,8 +213,8 @@ async function runWordPressMode(
 	php.mount(projectPath, documentRoot);
 	if (!php.fileExists(`${documentRoot}/wp-config.php`)) {
 		await initWordPress(php, 'user-provided', documentRoot, absoluteUrl);
+		copySqlite(projectPath);
 	}
-	copySqlite(projectPath);
```

Comment:
I noticed one thing here - we could use `mountSqlite` instead of `copySqlite` as mounting is enough to make it work in the 'wordpress' mode too.

However, if we change it, checking if `wp-config.php` exists won't be enough to decide whether to mount it. We should mount it only if the project was initialized with SQLite. Thoughts?

Reply:
Good call, let's close in favor of https://github.com/WordPress/wordpress-playground/issues/327